recipes/pyre2/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pyre2" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/andreasvc/pyre2/archive/v{{ version }}.tar.gz
+  sha256: 287f75e1612d4540625c9c6fea88adb26bcd4e1e83cf74e8bbef1b019a57f332
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
+  skip: true  # [py<36]
+
+requirements:
+  build:
+    - "{{ compiler('cxx') }}"
+    - cython
```

Comment:
You probably don't need cython here, on in host should be enough.

Reply:
Lemme test again in a conda env, but I think that's required at build time (since the .cpp module src is not packaged).

Reply:
Host is available at build time and make it cross-compile-able. We only put things in build that cannot be in host.

Reply:
Works fine in fresh conda-forge env (cython not required under build).  The next version will most likely have cmake and pybind11 deps.  Thanks!